locations/spiders/bakers_delight.py

```diff
@@ -0,0 +1,26 @@
+from scrapy.spiders import SitemapSpider
+
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class BakersDelightSpider(SitemapSpider, StructuredDataSpider):
+    name = "bakers_delight"
+    item_attributes = {"brand": "Bakers Delight", "brand_wikidata": "Q4849261"}
+    allowed_domains = ["www.bakersdelight.com.au"]
+    sitemap_urls = ["https://www.bakersdelight.com.au/wpsl_stores-sitemap.xml"]
+    sitemap_rules = [(r"/bakery-locator/", "parse_sd")]
+
+    def post_process_item(self, item, response, ld_data, **kwargs):
+        data_raw = response.xpath('//script[@id="wpsl-js-js-extra"]/text()').extract_first()
```

Comment:
Why not parse this as JSON instead of string-matching?

Reply:
Thanks, fixed. I'm not sure why I missed that.